airbyte-cdk/python/unit_tests/sources/declarative/stream_slicers/test_list_slicer.py

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import pytest as pytest
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.declarative.stream_slicers.list_stream_slicer import ListStreamSlicer
+
+
+@pytest.mark.parametrize(
+    "test_name, slice_values, slice_definition, expected_slices",
+    [
+        (
+            "test_single_element",
+            ["customer", "store", "subscription"],
+            {"owner_resource": "{{ slice_value }}"},
+            [{"owner_resource": "customer"}, {"owner_resource": "store"}, {"owner_resource": "subscription"}],
+        ),
+    ],
+)
+def test_substream_slicer(test_name, slice_values, slice_definition, expected_slices):
```

Comment:
is it useful to also have a test for the case where the list comes from the config like github?

Reply:
yes, done